### PR TITLE
Adding a react-canvas-lite package to build setup

### PR DESCRIPTION
**Pull request by zplata**
Building on the new `@rive-app/canvas-lite` package, we should add a complimentary package that uses this new dependency for React in a new `@rive-app/react-canvas-lite`.

Added a Rive Text example to Storybook examples to test rivs should still render (without Rive Text)

### CODE REVIEW

File: examples/stories/StateMachineDocs.stories.mdx [hunk 1]
```diff
@@ -165,6 +165,29 @@ Unlike the boolean and number inputs, you invoke the `.fire()` method on a trigg
   </Story>
 </Canvas>
 
+## Rive Text
```
Comment (HayesGordon) [inline examples/stories/StateMachineDocs.stories.mdx:168]: So this will not render anything?

File: examples/stories/StateMachineDocs.stories.mdx [hunk 2]
```diff
@@ -165,6 +165,29 @@ Unlike the boolean and number inputs, you invoke the `.fire()` method on a trigg
...
+
+To listen for Rive Events reported during state machine play, use the `on` API to add an event listener.
```
Comment (HayesGordon) [inline examples/stories/StateMachineDocs.stories.mdx:170]: Think this section needs to be updated, it's talking about events

File: npm/react-canvas-lite/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# @rive-app/react-canvas-lite
+
+Output for `rive-react` using the backing `@rive-app/canvas-lite` JS runtime.
+
+## Why Lite?
+
+The current `@rive-app/react-canvas` dependency supports all Rive features and contains the necessary backing dependencies to render those graphics. This `lite` version has the same API, but does not compile and build with certain dependencies in order to keep the package size as small as possible.
+
+At this time, this lite version of `@rive-app/react-canvas-lite` will not render [Rive Text](https://help.rive.app/editor/text) onto the canvas. Note however, that even if your Rive file may include Rive Text components, rendering the graphic should not cause any app errors, or cease to render.
```
Comment (HayesGordon) [inline npm/react-canvas-lite/README.md:9]: All that would happen is the text won't render right?

**Review by HayesGordon — APPROVED** [review]
LGTM! Just left some comments about the example

File: npm/react-canvas-lite/README.md (hunk 3 above)
Comment (zplata) [reply npm/react-canvas-lite/README.md:9]: That's right! Everything else in the hierarchy should still display/animate. Can make this clearer here

File: examples/stories/StateMachineDocs.stories.mdx (hunk 2 above)
Comment (zplata) [reply examples/stories/StateMachineDocs.stories.mdx:170]: Oop, good catch

File: examples/stories/StateMachineDocs.stories.mdx (hunk 1 above)
Comment (zplata) [reply examples/stories/StateMachineDocs.stories.mdx:168]: It will still render the graphic (all of the shapes/animated content without Rive text)

**Pull request merged**